tests/cypress/plugins/imageGenerator/addPlugin.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+    value: true
+});
```

Comment:
Hm, I've read about this and as I understand it is need in order to CommonJS modules were able to correctly import ES6 modules. Could you please explain me why we need it here?

As I see both this module and module where you import it, are CommonJS

Reply:
Initially, I compiled the file using the babel.  These lines are not needed. Removed.